modules/issue_tracker/css/issue_tracker.css

```diff
@@ -0,0 +1,116 @@
+.img-checkpic {
```

Comment:
This whole file seems to have a lot that shouldn't be there and looks like it's related to the imaging_browser, not the issue tracker. For instance, I doubt that the issue_tracker needs styling for ".img-checkpic" or ".mri-third-row-panel"